sermant-integration-tests/tag-transmission-test/servlet56-server-demo/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>io.sermant.tagtransmission</groupId>
+    <artifactId>servlet56-server-demo</artifactId>
+    <version>1.0.0</version>
```

Comment:
<parent>
        <artifactId>tag-transmission-test</artifactId>
        <groupId>io.sermant.tagtransmission</groupId>
        <version>1.0.0</version>
    </parent>

make this demo as the child pom of tag-transmission-test

Reply:
done

Reply:
> done

It seems that this change is difficult, and I need to override all the dependencies of the parent project on the introduction of springboot, probably more than 30, so that I can get the demo project to start successfully. 

Since the parent project relies on Spring2.x, Jakarta requires tomcat10, which in turn corresponds to Springboot3.x, is this necessary?

Reply:
ok, you can use profile id to controll complile profile，like

        <profile>
            <id>grpc-test</id>
            <modules>
                <module>grpc-client-demo</module>
                <module>grpc-server-demo</module>
                <module>grpc-api-demo</module>
                <module>tag-transmission-util-demo</module>
            </modules>
        </profile>

This can resolve CI/Checkstyle problem

Reply:
try @SuppressWarnings("uncommentedmain")